src/test/java/com/epam/tc/hw3/page/IndexPageMainContainer.java

```diff
@@ -0,0 +1,86 @@
+package com.epam.tc.hw3.page;
+
+import java.util.List;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+
+public class IndexPageMainContainer {
+
+    @FindBy(className = "benefit-icon")
+    private List<WebElement> listOfImages;
+
+    @FindBy(className = "icon-practise")
+    private WebElement iconPractise;
+
+    @FindBy(className = "icon-custom")
+    private WebElement iconCustom;
+
+    @FindBy(className = "icon-multi")
+    private WebElement iconMulti;
+
+    @FindBy(className = "icon-base")
+    private WebElement iconBase;
+
+    @FindBy(className = "benefit-txt")
+    private List<WebElement> listOfTexts;
+
+    @FindBy(xpath = "//div[normalize-space(span) = 'To include good practicesand ideas from successfulEPAM project']")
```

Comment:
the same thing which you did in EX2, you don't need these elements, just one and verify that elements in list has the text

Reply:
removed excessive FindBy annotations with texts, and now a list of WebElements is used in "for" loop with getText method